modules/prediction/common/prediction_system_gflags.cc

```diff
@@ -32,6 +32,8 @@ DEFINE_string(prediction_data_dir,
               "Prefix of files to store feature data");
 DEFINE_string(offline_feature_proto_file_name, "",
               "The bin file including a series of feature proto messages");
+DEFINE_string(extract_feature_type, "",
```

Comment:
I don't know if it's better to use boolean for such 2-choose-1 stuff, rather than use string.

Reply:
I think it is okay to use a string as we might support more feature type in future.